lib/platform_android/network_service_impl.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:flutter/material.dart';
+import 'package:registration_client/platform_spi/network_service.dart';
+import 'package:http/http.dart' as http;
+
+class NetworkServiceImpl implements NetworkService {
+  @override
+  Future<String> checkInternetConnection() async {
+
+    try {
+      final response = await http.get(Uri.parse(
+          'https://api-internal.dev2.mosip.net/v1/syncdata/actuator/health'))
```

Comment:
make this configurable from parameter, while qa-triad env is called it'll refer to dev2

Reply:
Using it on flutter side so added it with the constants. For qa-triad api url, it is declared as baseUrl in build.gradle file as it is used on android side.

Reply:
presently we are picking it from app_config.dart, we can replace it's location in future